pages/exercises/[id].js

```diff
@@ -12,28 +12,48 @@ export default function DetailsPage({ exercises }) {
   const exercise = exercises.find((exercise) => exercise.id === id);
   return (
     <>
-      <ExerciseDetailsHeading>
-        <StyledBackLink href={"/"}>←</StyledBackLink>
-        <h1>{exercise.name}</h1>
-      </ExerciseDetailsHeading>
+      <HeaderContainer>
+        <StyledBackLink href={"/"}>
+          <img src="/icons/back-arrow-white.svg" alt="Back" />
+        </StyledBackLink>
+      </HeaderContainer>
       <ExerciseDetailsContainer>
         <ExerciseDetails exercise={exercise} />
       </ExerciseDetailsContainer>
     </>
   );
 }
+
 const StyledBackLink = styled(Link)`
   text-decoration: none;
-  color: black;
-  font-size: 3rem;
+  background-color: #FAC74E;
```

Comment:
Since your userstory wants to define global stylings you should better define global color variables and use them instead of defining hex codes.